src/hct_mis_api/apps/household/fixtures.py

```diff
@@ -557,3 +557,51 @@ def create_individual_document(individual: Individual, document_type: Optional[s
         additional_fields["type"] = document_type
     document = DocumentFactory(individual=individual, **additional_fields)
     return document
+
+
+def generate_additional_doc_types() -> None:
```

Comment:
@pavlo-mk  this is what we would like to have as fixture!

Reply:
we are generating this one only for local fake data not for unit tests